.github/workflows/cd.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish logstash-oss-with-opensearch-plugin
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  Build:
+    name: Build and Install logstash-output-opensearch
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout plugin
+        uses: actions/checkout@v1
+
+      - name: Build plugin
+        run: |
+          gem build logstash-output-opensearch.gemspec
+
+      - name: Download and extract logstash
+        run: |
+          mkdir -p prepare-install;
+          mv logstash-output-opensearch-*.gem prepare-install/logstash-output-opensearch.gem;
+          wget --no-verbose https://artifacts.elastic.co/downloads/logstash/logstash-oss-7.13.2-linux-x86_64.tar.gz -P prepare-install/;
+          cd prepare-install;
+          tar xzvf logstash-oss-7.13.2-linux-x86_64.tar.gz;
+
+
+      - name: Install plugin
+        run: |
+          cd prepare-install;
+          logstash-7.13.2/bin/logstash-plugin install logstash-output-opensearch.gem;
+          logstash-7.13.2/bin/logstash-plugin list;
+          tar -czf logstash-oss-with-opensearch-plugin-7.13.2-linux-x86_64.tar.gz logstash-7.13.2
+
+      - name: Upload logstash-oss-with-opensearch-plugin
+        uses: actions/upload-artifact@v2
+        with:
+          name: "logstash-oss-with-opensearch-plugin-7.13.2-linux-x86_64.tar.gz"
+          path: prepare-install/logstash-oss-with-opensearch-plugin-7.13.2-linux-x86_64.tar.gz
```

Comment:
For discussion, should the artifact use `x64` as this is what we use for OpenSearch artifacts: https://opensearch.org/downloads.html or do we want to keep consistent with logstash convention of `x86_64`

Reply:
I renamed to our naming convention